static/app/views/organizationGroupDetails/grouping/grouping.tsx

```diff
@@ -325,17 +320,28 @@ const Header = styled('p')`
   }
 `;
 
-const Footer = styled('p')`
-  && {
-    margin-top: ${space(2)};
-  }
-`;
-
 const Body = styled('div')`
   display: grid;
   gap: ${space(3)};
 `;
 
+const Actions = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr max-content;
+  align-items: center;
+`;
```

Comment:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/9060071/165109739-cb5e2a86-f92b-4608-83f0-884ccff5b6ba.png">
Should we add a bit more space here?

Reply:
oh yes, I missed this...will do..thanks!

Reply:
fixed in https://github.com/getsentry/sentry/pull/33858/commits/443d7fd5fb60d19a199778769310f4ad997636f4